lib/jsonapi/rails/renderer.rb

```diff
@@ -33,6 +33,7 @@ def rails_renderer(renderer)
         reverse_mapping = request.env[ActionController::REVERSE_MAPPING_KEY]
         options = options.merge(_reverse_mapping: reverse_mapping)
         json = renderer.render(json, options) unless json.is_a?(String)
+        json = json.to_json unless json.is_a?(String)
```

Comment:
This works for the master version of `jsonapi-serialization`, but doesn't with the latest released one. This keeps the gems more independent.